Tests/SonarScanner.MSBuild.PreProcessor.Test/RoslynAnalyzerProviderTests.cs

```diff
@@ -78,21 +82,158 @@ public void RoslynConfig_ValidProfile()
     {
         var sonarProperties = new ListPropertiesProvider(new Dictionary<string, string>
         {
-            // for ruleset
+            {"wintellect.pluginKey", "wintellect"},
+            {"wintellect.pluginVersion", "1.13.0"},
+            {"wintellect.staticResourceName", "wintellect.zip"},
+            {"sonar.cs.analyzer.dotnet.pluginKey", "csharp"},
+            {"sonar.cs.analyzer.dotnet.pluginVersion", "1.42.0"},
+            {"sonar.cs.analyzer.dotnet.staticResourceName", "SonarAnalyzer.zip"},
```

Comment:
This should have also vbnet in the scaffolding, that will not make it to the output